tests/test.test.ts

```diff
@@ -0,0 +1,4 @@
+describe('Calculator', () => {
```

Comment:
Accidentally committed this?

Reply:
Here's a sample of test implementation for CI.

Reply:
Aha, I see it in the CI. Alright.